src/DIRAC/FrameworkSystem/scripts/dirac_populate_component_db.py

```diff
@@ -163,7 +160,7 @@ def main():
                         "Systems/"
                         + availableDB[db]["System"]
                         + "/"
-                        + cfg.getOption("DIRAC/Setups/" + CSGlobals.getSetup() + "/" + availableDB[db]["System"])
+                        + cfg.getOption(availableDB[db]["System"])
```

Comment:
This line should be just dropped

Reply:
probably the whole script can be dropped.